content/en/docs/Concepts/operators-on-cluster.md

```diff
@@ -0,0 +1,21 @@
+# When to use operators on a cluster
+
+
+Operators are good for automating the knowledge of how to operate a complex system. Operators are a high-privilege component and by design, they run persistently inside your cluster. An operators can handle features like automatic scaling in response to load, backup and restore, integration with metric systems like Prometheus, and even failure detection and automatic tuning in response to usage patterns.
```

Comment:
Why not just use existing descriptions of what operators are from existing sources?

https://operatorframework.io/ see "What is an Operator" paragraph.

https://docs.openshift.com/container-platform/4.7/operators/understanding/olm-what-operators-are.html